tests/e2e/default/contacts/contact-details.wdio-spec.js

```diff
@@ -69,10 +87,18 @@ describe('Contact details page', () => {
       expect(await (await contactPage.rhsReportListElement()).isDisplayed()).to.equal(true);
       expect(await (await contactPage.rhsTaskListElement()).isDisplayed()).to.equal(true);
 
-      expect((await contactPage.getAllRHSReportsNames()).length).to.equal(1);
-      expect((await contactPage.getAllRHSTaskNames()).length).to.deep.equal(1);
+      expect((await contactPage.getAllRHSReportsNames()).length).to.equal(DOCS_DISPLAY_LIMIT);
+      expect((await contactPage.getAllRHSTaskNames()).length).to.deep.equal(DOCS_DISPLAY_LIMIT);
     });
 
+    it(
+      'should show contact summary that has the full context for reports > 50' +
+      'validate that the pregnancy card is always displayed', async () => {
+
+        expect(await contactPage.pregnancyCard().isDisplayed()).to.be.true;
```

Comment:
At least we should check for 3 fields to ensure this card is taking the reports passed.
Fields like: `High risk`, `completed ANC clinic`, `current danger signs`